packages/avs-bot/src/main.ts

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+import * as path from 'path';
+import {FileEngine} from '@wireapp/store-engine-fs';
+import {Account} from '@wireapp/core';
+import {APIClient} from '@wireapp/api-client';
+import {ClientType} from '@wireapp/api-client/dist/client';
+import {
+  AUTH_TABLE_NAME,
+  AUTH_COOKIE_KEY,
+  AUTH_ACCESS_TOKEN_KEY,
+  AccessTokenData,
+  Cookie,
+  LoginData,
+} from '@wireapp/api-client/dist/auth';
+import {CRUDEngine} from '@wireapp/store-engine';
+import {PayloadBundleType} from '@wireapp/core/dist/conversation';
+import {Avs} from './avs';
+
+const wrtc = require('wrtc');
+
+declare global {
```

Comment:
IMHO this is AVS related as well, can it be moved to the avs file as well?

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/16edff26574f9ab4f1e8b56685ea1dabf52dedd7